cpp/oneapi/dal/backend/primitives/selection/kselect_by_rows_quick.hpp

```diff
@@ -34,6 +34,8 @@ namespace oneapi::dal::backend::primitives {
 // Performs k-selection using Quick Select algorithm which is based on row partitioning
 template <typename Float>
 class kselect_by_rows_quick : public kselect_by_rows_base<Float> {
+    static constexpr Float max_float = oneapi::dal::detail::limits<Float>::max();
```

Comment:
Just one usage, the function can be used directly in place

Reply:
Was discussed offline